rita/src/rita_common/dashboard/mod.rs

```diff
@@ -77,6 +78,7 @@ impl Handler<GetOwnInfo> for Dashboard {
                             balance: balance
                                 .to_i64()
                                 .ok_or(OwnInfoError::DownCastError(balance))?,
+                            local_fee: 0,
```

Comment:
why bother putting this here is it's going to be hardcoded?

Reply:
Yup, left it by accident, thanks